demos/whisper-base/whisper.js

```diff
@@ -238,18 +341,54 @@ export class Whisper {
             attention_mask = new ort.Tensor("int32", new Int32Array(4).fill([1, 1, 1, 1]), [1, 4]);
         }
         // create decoder input for the first inference
-        const decoder_input = {
+        const decoder_inputs = {
             input_ids: new ort.Tensor("int32", new Int32Array(tokens), [1, 4]),
             attention_mask: attention_mask,
-            encoder_hidden_states: last_hidden_state,
+            encoder_hidden_states: encoder_outputs.last_hidden_state,
         };
         // console.log(`Non-KV cache decoder input preparation time: ${(performance.now() - start).toFixed(2)}ms`);
         // start = performance.now();
         // run the first inference which generates SA and CA KV cache
-        const decoder_output = await this.models["decoder"]["sess"].run(decoder_input);
+        // Create fetches object with pre-allocated outputs for IO binding
```

Comment:
By "fetches", do you mean "inputs"? In the context of WebNN code, inputs would make more sense, as "fetches" is an obscure ORT term.

```suggestion
      // Create inputs object with pre-allocated outputs for IO binding
```

Reply:
Done.